src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* **Breaking change** `http.target` replaced with `http.route`
+([#3897](https://github.com/open-telemetry/opentelemetry-dotnet/issues/3897))
```

Comment:
```suggestion
([#3903](https://github.com/open-telemetry/opentelemetry-dotnet/issues/3903))
```

Reply:
well then it should be /pull, not /issue